src/index.html

```diff
@@ -6,7 +6,6 @@
     <title>Sleep Outside | Home</title>
     <link rel="stylesheet" href="/css/style.css" />
     <script src="js/main.js" type="module"></script>
-    <script src="js/product-listing.js" type="module"></script>
```

Comment:
You should never have more than one Javascript file included within an HTML file. It just causes pain. If you need stuff from a different JS file, import it into the single file that you include (main.js in this case).